libs/components/split-view/src/lib/modules/split-view/split-view-media-query.service.ts

```diff
@@ -31,7 +31,7 @@ export class SkySplitViewMediaQueryService {
     });
   }
 
-  public setBreakpointForWidth(width: number): void {
+  public setBreakpointForWidth(width: number | undefined): void {
     let breakpoint: SkyMediaBreakpoints;
 
     if (this.isWidthWithinBreakpiont(width, SkyMediaBreakpoints.xs)) {
```

Comment:
This isn't the fault of this PR, but I couldn't help but notice this method is misspelled: `isWidthWithinBreakpiont`.

Reply:
Fixed.